src/main/kotlin/com/github/hanseter/json/editor/validators/ValidationEngine.kt

```diff
@@ -135,7 +136,10 @@ object ValidationEngine {
 
     private fun createErrorMessage(subErrors: Int, errors: List<String>?): List<String>? {
         if (subErrors < 1) return errors
-        val subErrorMessage = listOf("$subErrors sub-error" + if (subErrors > 1) "s" else "")
+        val subErrorMessage =
+            if (subErrors == 1) listOf("1 ${JsonPropertiesMl.bundle.getString("jsonEditor.validators.subError")}")
+            else listOf("$subErrors ${JsonPropertiesMl.bundle.getString("jsonEditor.validators.subErrors")}")
+
```

Comment:
Please use a message format in case there's a language in which the number does not occur in the first position.